pkg/config/specification.go

```diff
@@ -132,6 +138,8 @@ func init() {
 	viper.SetDefault("backendFlushInterval", "20ms")
 	viper.SetDefault("requestID", true)
 
+	viper.SetDefault("respondingTimeouts.IdleTimeout", 180*time.Second)
```

Comment:
should not it be `respondingTimeouts.idleTimeout`? `idle` not capital - test is the best way to check it =)

Reply:
Both work, but I'll add a test for it anyways